god-mode.el

```diff
@@ -189,7 +190,7 @@ If it was not active when `god-local-mode-pause' was called, nothing happens."
         (execute-kbd-macro binding)))))
 
 (define-key universal-argument-map (kbd "u")
-  #'god-mode-maybe-universal-argument-more)
+	    #'god-mode-maybe-universal-argument-more)
```

Comment:
This indentation seems off. Can it be reverted?

Reply:
It was the result of applying `indent-region`. But I can revert it back to what it was before